### PR TITLE
Frontiersiliconradio binding for internet radios based on frontier silicon chipset

### PR DESCRIPTION
**Pull request by paphko**
Based on pull request #1921, I revised the code, realized comments from Thomas, added a cache period (similar to Anel and NH bindings), and successfully tested it with my Medion MD87180 radio.

### CODE REVIEW

**Comment by teichsta** [comment]
Thanks @paphko. I didn't went through the code already. Did you incorporate all my feedback? Best, Thomas E.-E.

**Comment by paphko** [comment]
...almost :-) I missed this list and I will add it as soon as I find some time:
* please make sure each of you classes contains an appropriate description on class level and has all public methods documented
* update license header (using mvn license plugin)
* add your binding to distribution/pom.xml
* add your binding to bundles/binding/pom.xml

**Comment by paphko** [comment]
...done!

**Comment by paphko** [comment]
I finished all open issues. Anything else I can do?

**Comment by teichsta** [comment]
no, i'll take care of your binding in the course of the week …

**Comment by teichsta** [comment]
is there a wiki page available already?

**Comment by paphko** [comment]
@teichsta you're right, there is no wiki page yet. I'll add one next weekend.

**Comment by teichsta** [comment]
great, thanks!

File: bundles/binding/org.openhab.binding.frontiersiliconradio/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ org.openhab.core.library.items,
+ org.openhab.core.library.types,
+ org.openhab.core.types,
+ org.openhab.model.item.binding,
+ org.osgi.framework,
+ org.osgi.service.cm,
+ org.osgi.service.component,
+ org.osgi.service.event,
+ org.slf4j
+Export-Package: org.openhab.binding.frontiersiliconradio
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.frontiersiliconradio/META-INF/MANIFEST.MF]: remove export unless you really need this

**Comment by teichsta** [comment]
PR looks good to me. Could you finally please squash your commits into one. Then i would be happy to merge into master. Thanks, Thomas E.-E.

**Comment by paphko** [comment]
@teichsta thanks for the new review. I squashed all commits into one, I hope that is how you want it?
Btw, I noticed that the author of this commit is the author of the first squashed commit. So Rainer is the only remaining author in the git history...

**Comment by paphko** [comment]
Wiki page finished: https://github.com/openhab/openhab/wiki/Frontier-Silicon-Radio-Binding

**Comment by teichsta** [comment]
great, thanks!

**Comment by paphko** [comment]
There will be a new version of this binding for ESH: https://github.com/eclipse/smarthome/pull/429

**Pull request merged**